mission-vision-values.md

```diff
@@ -0,0 +1,99 @@
+# OTel Mission, Vision, and Values
+
+## Mission (our overall north star as a community)
+
+OpenTelemetry's Mission: **to enable effective observability through
```

Comment:
Other options from the gdoc – I welcome suggestions here!

```
to make high-quality telemetry a built-in feature for all cloud-native software.
to enable high-quality observability through ubiquitous, portable, high-quality telemetry
to enable effective observability through built-in, portable, open-source telemetry
to enable effective observability by making high-quality telemetry a built-in feature for all cloud-native software
to enable effective observability through turnkey telemetry
to enable effective observability through high-quality, turnkey telemetry
to enable observability through high-quality, turnkey, and ubiquitous telemetry
to enable observability through high-quality, open source telemetry
to enable observability through high-quality telemetry
to enable excellent observability through high-quality telemetry
```

Reply:
I like "to make high-quality telemetry a built-in feature for all cloud-native software." 

To me the defining feature of our mission is to get frameworks to build in first party support. It's what makes us most different from other solutions.

Reply:
what @dyladan said, we need "cloud-native" to be our primary remit (but also we shouldn't exclude other users).

wrt including "observability" or not, I think the telemetry is a crucial ingredient, but adopting otel alone won't give you o11y

Reply:
I think I wrote `to make high-quality telemetry a built-in feature for all cloud-native software` and so can still get behind it personally. :)

The critique – from @yurishkuro IIRC – was:

>This to me reads more like a Vision as defined in the About section, an "ideal state", and as such not appropriate as Mission because then mission just says "implement Vision"

In order to avoid the tautological aspect, I was trying to do a sort of "five whys" exercise and ask myself what high-quality telemetry is *good* for, hence **enabling** observability (rather than **being** o11y, @lizthegrey). Maybe that distinction is too subtle, though.

If we don't want to say "observability" here, are we able to zoom out and articulate a larger purpose for OpenTelemetry beyond, well, telemetry?

Separately regarding "cloud-native": as a litmus test, would we welcome the news that old-school networking hardware is going to support OTLP export of load metrics? I would... but that's not necessarily "cloud-native." Of course OTel does *include* cloud-native, but do we want to be *exclusively* cloud-native? And if not, why is it in the mission?

Reply:
I think the current text is confusing Vision (ideal state) and Mission (how to get there). Vision come first.

Reply:
My vote is for: "to enable high-quality observability through ubiquitous, portable, high-quality telemetry"  for the reason mentioned by @bhs above.

 I would like to see OTel be adopted ubiquitously with no implication that its a "cloud thing". For me explicitly adding "cloud-native" is problematic.  I see a challenge that initiatives like OTel can meet is providing that ubiquity between cloud and non-cloud use cases. Ubiquity is the key principle for me. 

It may be semantics but As some one who worries about both worlds, being able to reason about observability as a singular thing across multiple domains is a big strength that I would encourage highlighting rather than ring fencing with the term "cloud-native"